app/controllers/users_controller.rb

```diff
@@ -16,15 +23,48 @@ def create
     if @user.save
       log_in @user
       flash[:success] = t "controllers.users.welcome"
-      redirect_to @user
+      redirect_back_or user
     else
       flash[:danger] = t "controllers.users.acc_reg_fail"
       render :new
     end
   end
 
+  def edit
+    @user = User.find_by id: params[:id]
```

Comment:
you need to define a function
def load_user
 @user = User.find_by id: params[:id]
end
and use before_action to call it

Reply:
done